components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -24,6 +24,7 @@ class ToolbarPresenter(
     fun start() {
         session = sessionManager.selectedSession
 
+        sessionManager.all.forEach { onSessionAdded(it) }
```

Comment:
Here I also do not understand why we loop over all sessions and then display the state based on the last custom tab we saw (in onSessionAdded()) what may or may not be what we actually display.

This probably has the same problem as EngineViewPresenter: I think we actually want to display the currently selected session (main browser activity) or a specific session (custom tab activit**ies**)